### ISSUE TITLE
Please do not include libpng16.a and libz.a within sources

### ISSUE THREAD

**Issue by dr-br** [jabcode/jabcode#11]
Compilation with new versions of gcc / os(?) is not possible due to relocation errors.
Instead of providing libpng and libz within the lib folder, simply use the one provided by the distro and modify Makefiles accordingly.

E.g. ubuntu:
sudo apt install libpng-dev zlib1g-dev

**Comment by jabcode** [comment]
You are right. The given libs are only for Ubuntu and Windows, compiled with gcc 4.8.4. If you are using other os, e.g. Mac, they have to be installed/compiled separately. We will consider your suggestion in the new version.
